whois-rpsl/src/main/java/net/ripe/db/whois/common/rpsl/RpslObject.java

```diff
@@ -108,6 +109,11 @@ public ObjectType getType() {
         return attributes;
     }
 
+    public void removeAttribute(AttributeType type) {
```

Comment:
RpslObject is meant to be immutable. Please call instead from SingleUpdateHandler:
RpslObject updatedObject = new RpslObjectBuilder(rpslObject).removeAttributeType(AttributeType.CHANGED).get();

Reply:
Changed. RpslObject is reverted to state before my changes